using-ide.md

```diff
@@ -230,10 +230,10 @@ You can also define custom templates in **Settings > Editor > Live Templates**.
 **Hot Reload**
 
 On Linux (keymap _Default for XWin_) and Windows the keyboard shortcuts
-are `ctrl-alt-;` and `ctrl-\`.
+are `ctrl`+`alt`+`;` and `ctrl`+`\`.
```

Comment:
Please use <kbd>Control</kbd>+<kbd>Alt</kbd>  and
                    <kbd>Control</kbd>+<kbd>Backslash</kbd>